playbooks/roles/ocim/meta/main.yml

```diff
@@ -10,6 +10,8 @@ dependencies:
     NGINX_PROXY_SERVER_NAME: "{{ OPENCRAFT_DOMAIN_NAME }}"
     NGINX_PROXY_SEND_TIMEOUT: 600s
     NGINX_PROXY_READ_TIMEOUT: 600s
+    NGINX_PROXY_ORIGIN_REGEX: '^(http|https)://.*\.opencraft\.com'
```

Comment:
Should I move this constant to https://github.com/open-craft/ansible-playbooks/blob/777edcf4cd224e446c0a631318bd48b684598711/playbooks/roles/ocim/defaults/main.yml ?